src/js/game/camera.js

```diff
@@ -484,6 +485,7 @@ export class Camera extends BasicSerializableObject {
         if (!this.checkPreventDoubleMouse()) {
             return;
         }
+        this.root.app.inputMgr.handleKeyup(event);
```

Comment:
Whats the reason you are manually calling keyup here?

Reply:
Because I haven't found how to bind it correctly that time
Fixed.